### PR TITLE
[DPE-2477] Add a minimal set of alert rules from the Awesome Prometheus Alert Rules blog

### PR DESCRIPTION
**Pull request by shayancanonical**
## Issue
We would like to import a minimal set of alert rules from the [Awesome Prometheus Alert Rules blog](https://samber.github.io/awesome-prometheus-alerts/rules#mysql)

## Solution
Import rules (noting slight modifications). Do not include any replication rules as they are not Group Replication related

image

### CODE REVIEW

File: src/alert_rules/prometheus/metrics_alert_rules.yaml [hunk 1]
```diff
@@ -11,3 +12,65 @@ groups:
...
 
+      # 2.1.2
+      # customized: 80% -> 90%
+      - alert: MySQLTooManyConnections(>80%)
+        expr: max_over_time(mysql_global_status_threads_connected[1m]) / mysql_global_variables_max_connections * 100 > 90
+        for: 2m
+        labels:
+          severity: warning
+        annotations:
+          summary: MySQL too many connections (> 90%) (instance {{ $labels.instance }})
```
Comment (taurus-forever) [inline src/alert_rules/prometheus/metrics_alert_rules.yaml:23]: It is 80% in lines 17-18 and 90% here. Typos?

**Review by taurus-forever — COMMENTED** [review]
LGTM as a step 1, one typo inside otherwise LGTM. Thank you!

File: src/alert_rules/prometheus/metrics_alert_rules.yaml (hunk 1 above)
Comment (shayancanonical) [reply src/alert_rules/prometheus/metrics_alert_rules.yaml:23]: oops, sorry, yes typo. corrected in 029f4b826677a277e3f935c317413ecaacf108e4

**Review by paulomach — APPROVED** [review]
Ok for starting. We need to check if not having GR alerts/metrics is a show stopper though

**Pull request merged**